docs/dptp-triage-sop/sync-rover-group.md

```diff
@@ -0,0 +1,31 @@
+# Sync Rover Groups
+
+## How it works
+
+Rover groups are synced to the clusters by two jobs:
+
+- A [cronjob/sync-rover-groups-update](https://console-openshift-console.apps.ocp-c1.prod.psi.redhat.com/k8s/ns/ocp-test-platform/batch~v1~CronJob/sync-rover-groups-update/) on the psi cluster: It stores the groups from Rover to `ConfigMap/sync-rover-groups` in `namespace/ci` of `app.ci`.
+
+-  A periodic `[ProwJob/periodic-github-ldap-user-group-creator](https://github.com/openshift/release/blob/master/ci-operator/jobs/infra-periodics.yaml)`: It maintains the groups defined in `ConfigMap/sync-rover-groups` on the clusters.
```

Comment:
nit: you want the backticks inside the square brackets:

```markdown
[`ProwJob/periodic-github-ldap-user-group-creator`](https://github.com/openshift/release/blob/master/ci-operator/jobs/infra-periodics.yaml)
```

to get it rendered as a clickable link.

Reply:
Updated.